dnsviz/viz/dnssec.py

```diff
@@ -1793,8 +1793,8 @@ def _add_trust_to_orphaned_nodes(self, subgraph_name, trace):
             if INVIS_STYLE_RE.search(n.attr['style']) is not None:
                 continue
 
-            # if node is non-existent, then don't mark it, unless we are talking about an RRset
-            # or a non-existent trust anchor; it doesn't make sense to mark other nodes
+            # if node is nonexistent, then don't mark it, unless we are talking about an RRset
+            # or a nonexistent trust anchor; it doesn't make sense to mark other nodes
```

Comment:
I'm happy to drop these changes if they're problematic.

Reply:
These ones are fine.